src/seedu/addressbook/data/person/Name.java

```diff
@@ -37,10 +37,10 @@ public static boolean isValidName(String test) {
     }
 
     /**
-     * Retrieves a listing of every word in the name, in order.
+     * Retrieves a listing of every word in the name, in order and converts to lowercase.
      */
     public List<String> getWordsInName() {
-        return Arrays.asList(fullName.split("\\s+"));
+        return Arrays.asList(fullName.toLowerCase().split("\\s+"));
```

Comment:
I don't recommend putting change the name into lowercase here.. Better put it in the FindCommand class.